### ISSUE TITLE
Investigate git-fast-import exit code 1

### ISSUE THREAD

**Issue by LawnGnome** [sourcegraph/git-cvs-fast-import#25]
It looks like imports that don't actually import anything might result in exit code 1. I need to verify where that's coming from, what it means (since it's not in the manpage), and then special case as appropriate.

**Comment by LawnGnome** [comment]
It looks like this is actually the importer not recognising previously written commits in some cases:

```
warning: Not updating refs/heads/tags/head-version (new tip bd3f831f0d63cfcaa78c3bb5c49d7f7b5c45958a does not contain 17b10b30edf4628feea7363e081cba478a7f2085)
warning: Not updating refs/tags/head-version (new tip bd3f831f0d63cfcaa78c3bb5c49d7f7b5c45958a does not contain 17b10b30edf4628feea7363e081cba478a7f2085)
warning: Not updating refs/heads/tags/branch-b-version (new tip a84898ef39b6b72fa78e0694db9464c90bfe2e48 does not contain d906e3158bc715e40983b1328f3b91a5bf2fd3cc)
warning: Not updating refs/tags/branch-b-version (new tip a84898ef39b6b72fa78e0694db9464c90bfe2e48 does not contain d906e3158bc715e40983b1328f3b91a5bf2fd3cc)
```

**Comment by LawnGnome** [comment]
Specifically, the problem here is the fake commits we create for tags: we shouldn't recreate them if they haven't changed, and if they have, then we need to move forward rather than tag an unrelated commit.

**Comment by LawnGnome** [comment]
Fixed in #26.
